ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -10,6 +10,70 @@
     handle_numpy_out,
 )
 
+# global
```

Comment:
you are not supposed to import stuff in the middle of codes, and these are already imported at the start of this file.

Reply:
Done